src/shared/components/FlightOffer/FlightOffer.jsx

```diff
@@ -15,77 +19,44 @@ function FlightOffer({
   returnCarrier,
   direct,
 }) {
-  function watchFly() {
-    console.log('obserwuje');
+  const [active, setActive] = useState(false);
+  function addToFavourite() {
+    setActive(!active);
+    const quotes = getItemFromLocalStorage('quotes') || [];
+    const quote = {
+      departurePlace,
+      returnPlace,
+      price,
+      departureDate,
+      returnDate,
+      departureCarrier,
+      returnCarrier,
+      direct,
+    };
+    const theSamePrice = !!quotes.find((elem) => elem.price === price);
```

Comment:
This function should take id as an argument. You shouldn't do that.

Reply:
Fixed